testsuite/features/support/env.rb

```diff
@@ -26,13 +26,14 @@
 STARTTIME = Time.new.to_i
 Capybara.default_max_wait_time = ENV['CAPYBARA_TIMEOUT'] ? ENV['CAPYBARA_TIMEOUT'].to_i : 10
 DEFAULT_TIMEOUT = ENV['DEFAULT_TIMEOUT'] ? ENV['DEFAULT_TIMEOUT'].to_i : 250
+PROVIDER = ENV["PROVIDER"]
 
 # QAM and Build Validation pipelines will provide a json file including all custom (MI) repositories
 custom_repos_path = File.dirname(__FILE__) + '/../upload_files/' + 'custom_repositories.json'
 if File.exist?(custom_repos_path)
   custom_repos_file = File.read(custom_repos_path)
   $custom_repositories = JSON.parse(custom_repos_file)
-  $build_validation = true
+  build_validation = (PROVIDER != 'aws')
```

Comment:
Why??
build_validation is only true if we have a custom_repositories.json file, and for cloud, we will not have that...
So, it doesn't make sense to change this value.

Reply:
I'm using terracumber-cli with the option custom-repositories manage MU and repositories needed by uyuni.
When you use this variable, uyuni automaticatly set build_validation tags.
I add this variable so build validation is only set on internal environment.